Gemfile

```diff
@@ -8,6 +8,7 @@ gem 'autoprefixer-rails'
 gem 'awesome_print'
 gem 'aws-sdk-v1'    # remaining on v1 due to https://github.com/thoughtbot/paperclip/issues/1764
 gem 'bootstrap-sass'
+gem 'browser-timezone-rails'
```

Comment:
I hope it still works! :smile: 

We still use it but we are on rails 3.  Don't tell anyone!